content/cluster/_index.md

```diff
@@ -1,10 +1,81 @@
 ---
 title: "Cluster"
-date: 2019-06-18T16:42:20+02:00
-lastmod: 2019-06-18T16:42:20+02:00
-publishdate: 2019-06-18T16:42:20+02:00
+date: 2020-04-21T15:22:20+02:00
+lastmod: 2020-04-21T15:22:20+02:00
+publishdate: 2020-04-21T15:30:20+02:00
 draft: false
 weight: 13
 ---
 
-Nothing to see here (yet)
+# Move from 3 masters (control-plane) to 3 masters + workers cluster (control-plane + compute nodes)
+OCP4 allows to deploy control-plane only clusters or control-plane + compute nodes clusters. If you only deploy control-plane, the masters are labeled as workers as well, otherwise, they are labeled only as 'masters.
+
+If you want to add workers to a control-plane only cluster, you should do the following:
+
+## Create computes
+
+Once the computes are provisioned and ready as in:
+
+```
+oc get nodes
+```
+
+You need to label them as:
+```
+oc label nodes ocp4-worker-0.example.com node-role.kubernetes.io/worker="" 
+oc label nodes ocp4-worker-1.example.com node-role.kubernetes.io/worker=""
+oc get nodes
+```
+Note: you may want to remove the node-role.kubernetes.io="" label from the masters, up to you.
```

Comment:
node-role.kubernetes.io="" should be `node-role.kubernetes.io/worker="" `

Reply:
changed, thx!